### ISSUE TITLE
Request: Ascent Guidance: Variable Acceleration for Direct Insertion to Stable Orbit.

### ISSUE THREAD

**Issue by deckblad** [MuMech/MechJeb2#404]
When using ascent guidance, I like to decrease my max acceleration over time. I'll typically start at around 22m/s at launch. By the time I've matched my desired apogee, I'm at around 2m/s. The intent is to fall into a direct insertion; hitting my desired AP at the exact time I pass through AP while also bringing the PE up to match. All at once.

A chart as an example. For an insertion to 250km
0km  22m/s
10km 20m/s
20km 16m/s
40km 10m/s
80km 6m/s
120km 4m/s
200km 2m/s

That's a rough estimate... Can anyone with knowledge of orbital mechanics verify or provide a proper method for direct insertion?

My request could be one of two things:
1. A second adjustable curve on the Ascent Adjustment panel with three values: "Start Accel," "Ending Accel," and "Curve."
1. Perhaps there's a standard model equation for a direct insertion? If it's simple to emulate, it'd only require an additional check box on Ascent Guidance.

Thanks, guys.

**Comment by lamont-granquist** [comment]
Think I'm going to close this as you should pick one or the other.   Either you have realistic rockets so that they both cannot deep throttle and cannot relight, or you're in stock and you can insert a coast phase.  If you really want to model a Falcon 9 with a direct ascent then it should be done in RSS/RO or with stock-size planets and SMURFF at a minimum.

Direct insertion is covered under the PEG code or #1061 and since optimum exoatmospheric ascent profiles are provably 100% throttle and I just have no interest in coding this, I think I'm going to close this issue.

TL;DR:  stock is silly.
